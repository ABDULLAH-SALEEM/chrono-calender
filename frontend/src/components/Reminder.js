import React from "react";
const Reminder = () => (
  <div data-testid="reminder-root">Reminder Component</div>
);
export default Reminder;
