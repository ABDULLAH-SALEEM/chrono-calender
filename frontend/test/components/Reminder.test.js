import React from "react";
import { render, screen } from "@testing-library/react";
import Reminder from "../../src/components/Reminder";

describe("Reminder", () => {
  it("renders without crashing", () => {
    render(<Reminder />);
    expect(screen.getByTestId("reminder-root")).toBeInTheDocument();
  });

  it("matches snapshot", () => {
    const { asFragment } = render(<Reminder />);
    expect(asFragment()).toMatchSnapshot();
  });
});
