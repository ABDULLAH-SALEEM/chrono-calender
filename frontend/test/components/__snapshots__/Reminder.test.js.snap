// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reminder matches snapshot 1`] = `
<DocumentFragment>
  <div
    data-testid="reminder-root"
  >
    Reminder Component
  </div>
</DocumentFragment>
`;
